syft/pkg/binary.go

```diff
@@ -24,7 +24,10 @@ type ELFBinaryPackageNoteJSONPayload struct {
 	Architecture string `json:"architecture,omitempty"`
 
 	// OSCPE is a CPE name for the OS, typically corresponding to CPE_NAME in os-release (e.g. cpe:/o:fedoraproject:fedora:33)
-	OSCPE string `json:"osCPE,omitempty"`
+	OSCPE string `json:"osCpe,omitempty"`
```

Comment:
we should also no break current behavior -- adding a new field to https://github.com/anchore/syft/blob/47e1cee5a55b5d0ea63dc13a26b3d1acab2181e3/syft/pkg/cataloger/binary/elf_package_cataloger.go#L36 that also unmarshals the old `osCPE` field and reconciles that value to the `osCpe` (new) field would help here. Since this is a grey area instead of an addition schema-wise lets make this a revision (instead of `16.1.1` lets go to `16.2.0`)